spec/spec_helper.rb

```diff
@@ -1,7 +1,6 @@
 require 'puppetlabs_spec_helper/module_spec_helper'
 require 'rspec-puppet-facts'
-
-require 'spec_helper_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_local.rb'))
+require 'spec_helper_acceptance_local'
```

Comment:
`pdk update` will revert this:
```
diff --git a/spec/spec_helper.rb b/spec/spec_helper.rb
index 01f9db1..feb5720 100644
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,6 +1,7 @@
 require 'puppetlabs_spec_helper/module_spec_helper'
 require 'rspec-puppet-facts'
-require 'spec_helper_acceptance_local'
+
+require 'spec_helper_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_local.rb'))
 
 include RspecPuppetFacts 
```

spec_helper and spec_helper_local are for unit test setup&helpers, spec_helper_acceptance and spec_helper_acceptance_local are for acceptance test setup & helpers. Please do keep them separated. You can go back to the original files on master to figure out which helpers belong into which file.

Reply:
Think this should be resolved now @DavidS . @lionce had done most of this before signing off.